src/verifier/mem_encode/encoder.hpp

```diff
@@ -245,6 +245,12 @@ class encoder {
   template <class Iterator>
   void write_mols(Iterator begin, Iterator end, BinStrCursor &bsp,
                   VisitLogRef visited) {
+    BinStrCursor last_valid_bsp;
+    Iterator last_valid_it = end;
+    int first_func = 0;
+    CheckpointRef last_valid_checkpoint = NULL;
```

Comment:
ここは `nullptr` のほうが良いです
```suggestion
    CheckpointRef last_valid_checkpoint = nullptr;
```